src/main.ts

```diff
@@ -151,10 +163,10 @@ function customGenericWin(url: string, providedMenuTemplate: (MenuItemConstructo
 
 	// if hideAds is enabled, hide them. then show the window
 	genericWin.once('ready-to-show', () => {
-		if (userPrefs.hideAds) genericWin.webContents.insertCSS(hideAdsCSS);
+		if (userPrefs.blockAds) genericWin.webContents.insertCSS(hideAdsCSS);
```

Comment:
if you want to change a setting key that has been previously used, you have to gracefully migrate it - see what i do here:
https://github.com/KraXen72/crankshaft/blob/e9ebf984e60c3acb6d23e2a20ce72ff9e8bda247/src/main.ts#L60